Source/ConvolutionFunctions/arm_convolve_1x1_s8_fast.c

```diff
@@ -77,9 +76,13 @@ arm_cmsis_nn_status arm_convolve_1x1_s8_fast(const cmsis_nn_context *ctx,
 
     if (lhs_rows % 3 == 0 && rhs_rows % 2 == 0 && rhs_rows > lhs_rows * 4)
     {
-        int32_t buf[rhs_rows];
-        int32_t *ker_sum = buf;
-        int32_t *buf_start = buf;
+        if (ctx->buf == NULL)
+        {
+            return ARM_CMSIS_NN_ARG_ERROR;
+        }
+
+        int32_t *ker_sum = ctx->buf;
+        int32_t *buf_start = ctx->buf;
```

Comment:
Could you update the prototype of arm_nn_mat_mult_nt_t_fast_s8 that it has restrictions on the lhs_rows that it can handle?